packages/fx-core/resource/package.nls.json

```diff
@@ -217,13 +217,16 @@
   "plugins.spfx.error.invalidDependency": "Failed to validate package %s",
   "plugins.spfx.error.installDependency": "Failed to install package %s. Learn how to remediate in the [output](command:fx-extension.showOutputChannel).",
   "plugins.spfx.error.noConfiguration": "There's no .yo-rc.json file in your SPFx project, add the configuration file and try again.",
-  "plugins.spfx.error.devEnvironmentNotSetup": " SPFx development environment has not been set up correctly. Visit [Set up your SharePoint Framework development environment | Microsoft Learn](%s) to learn more.",
+  "plugins.spfx.error.devEnvironmentNotSetup": "SPFx development environment has not been set up correctly. Visit [Set up your SharePoint Framework development environment | Microsoft Learn](%s) to learn more.",
   "_plugins.spfx.error.devEnvironmentNotSetup.comment": "'Microsoft Learn' and 'SharePoint' are product brand names, which should not be localized. ",
   "plugins.spfx.scaffold.dependencyCheck": "Checking dependencies...",
   "plugins.spfx.scaffold.dependencyInstall": "Installing dependencies. This may take more than 5 minutes to finish.",
   "plugins.spfx.scaffold.scaffoldProject": "Generate SPFx project using Yeoman CLI",
   "plugins.spfx.scaffold.updateManifest": "Update webpart manifest",
   "plugins.spfx.GetTenantFailedError": "Cannot get tenant %s %s",
+  "plugins.spfx.error.installLatestDependencyError": "Encountered unknown issues when setting up SPFx environment in HOME/.fx folder. You could follow [Set up your SharePoint Framework development environment | Microsoft Learn](%s) to set up global SPFx environment.",
+  "_plugins.spfx.error.installLatestDependencyError.comment": "'Microsoft Learn' and 'SharePoint' are the product brand names and HOME/.fx is the folder path, which should not be localized.",
+  "plugins.spfx.error.scaffoldError": "Project creation failed. A possible reason could be from Yeoman SharePoint Generator.",
```

Comment:
From SPFx official doc, it should be 'Yeoman generator for the SharePoint Framework' or 'SPFx generator' https://learn.microsoft.com/en-us/sharepoint/dev/spfx/yeoman-generator-for-spfx-intro

Reply:
I pick the words from https://learn.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part#create-a-new-web-part-project "Create a new project by running the Yeoman SharePoint Generator"